crates/core/src/fnc/mod.rs

```diff
@@ -286,6 +286,8 @@ pub fn synchronous(
 		"not" => not::not,
 		//
 		"object::entries" => object::entries,
+		"object::extend" => object::extend,
+		"object::remove" => object::extend,
```

Comment:
Did you mean `object::remove`?

```suggestion
		"object::remove" => object::remove,
```

Reply:
Well spotted @rushmorem ! 
@Dhghomon the language test is now failing